docs/theia.md

```diff
@@ -0,0 +1,68 @@
+# Theia
+
+Theia is the command-line tool which provides access to Theia network flow
+visibility capabilities.
+
+## Table of Contents
+
+<!-- toc -->
+- [Installation](#installation)
+- [Compile](#compile)
```

Comment:
"Compilation" to be consistent with others.

Reply:
Thanks, addressed.